101-recovery-services-create-vm-and-configure-backup/README.md

```diff
@@ -0,0 +1,21 @@
+# Create simple windows VM and configure backup
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-recovery-services-create-vm-and-configure-backup%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-recovery-services-create-vm-and-configure-backup%2Fazuredeploy.json" target="_blank">
+    <img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+This template allows you to deploy simple Windows VM and Recovery Services Vault. Also configures Protection with DefaultPolicy

```

Comment:
This template allows you to deploy simple Windows VM and Recovery Services Vault configured with a default protection policy.